OurUmbraco/NotificationsCore/Notifications/ScheduleHangfireJobs.cs

```diff
@@ -52,6 +58,16 @@ public void UpdateMeetupStats()
             RecurringJob.AddOrUpdate(() => UpdateMeetupStatsJsonFile(), Cron.MinuteInterval(15));
         }
 
+        public void UpdateUpForGrabsIssues()
+        {
+            RecurringJob.AddOrUpdate(() => AddCommentToUpForGrabsIssues(), Cron.HourInterval(100));
```

Comment:
This will run each 100th hour. `Cron.MinuteInterval(100)` should make it every 100th minute instead.